Doc/howto/logging-cookbook.rst

```diff
@@ -1933,30 +1933,32 @@ This dictionary is passed to :func:`~config.dictConfig` to put the configuration
 
     LOGGING = {
         'version': 1,
-        'disable_existing_loggers': True,
+        'disable_existing_loggers': False,
         'formatters': {
             'verbose': {
-                'format': '%(levelname)s %(asctime)s %(module)s %(process)d %(thread)d %(message)s'
+                'format': '{levelname} {asctime} {module} {process:d} {thread:d} {message}',
+                'style': '{',
             },
             'simple': {
-                'format': '%(levelname)s %(message)s'
+                'format': '{levelname} {message}',
+                'style': '{',
             },
         },
         'filters': {
             'special': {
                 '()': 'project.logging.SpecialFilter',
                 'foo': 'bar',
-            }
+            },
+            'require_debug_true': {
```

Comment:
This filter is Django-specific, so I don't believe it should be included here.

Reply:
This example it documented as: _"Below is an example of a logging configuration dictionary - it's taken from the documentation on the Django project"_, that's why I wanted to keep it in sync, but I can remove `require_debug_true` if you prefer.

Reply:
Well, that preamble mentions Django as a form of credit/attribution - but the example should be otherwise standalone, in the sense of not requiring a reader unfamiliar with Django to have to wonder whether they need to look into what that filter is exactly.

Reply:
Don't get me wrong, I love Django!

Reply:
:green_heart: Updated :+1: